misc/README.md

```diff
@@ -7,6 +7,9 @@ The process for updating the `make e2e`'s input is manual.  Each time a go-algor
-| `make e2e` runs on every PR. Validate results on the next CI run.                                                                                                                                                                                                              | * Find the most recent workflow starting _after_ updating the S3 bucket:  [https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all](https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all).<br/>* If successful, confirm the test used the artifact you uploaded.  Click a job link, expand _make e2e_ and search for `INFO:util:s3://`.<br />* If unsuccessful, investigate source of error.  If additional help is needed, raise in #indexer.  It is _your_ responsibility to ensure the build is fixed. |
\ No newline at end of file
+| `make e2e` runs on every PR. Validate results on the next CI run.                                                                                                                                                                                                              | * Find the most recent workflow starting _after_ updating the S3 bucket:  [https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all](https://app.circleci.com/pipelines/github/algorand/indexer?branch=develop&filter=all).<br/>* If successful, confirm the test used the artifact you uploaded.  Click a job link, expand _make e2e_ and search for `INFO:util:s3://`.<br />* If unsuccessful, investigate source of error.  If additional help is needed, raise in #indexer.  It is _your_ responsibility to ensure the build is fixed. |
+
+# Help
+`e2elive.py` requires access to algorand-indexer and goal binaries. If you receive an error related to the algorand-indexer binary then you need to build the indexer from scratch by running `make` in the root of the indexer repository. This will insert a binary in the `cmd/algorand-indexer` directory, where `e2elive.py` expects it by default. <br>If the error is related to goal then the most likely cause is that the goal binary hasn't been installed as a package in the GOPATH system variable. Navigate to the go-algorand repository directory and run `make install`. If the error persists then run `export GOPATH=$(go env GOPATH)` followed by `export PATH=$PATH:$GOPATH/bin` in order to verify that the binary is accessible.</br>
```

Comment:
@algoidurovic Informational:  I leave the note to make sure you're aware.  Feel welcomed to resolve once read.

* Outside of a table, it's possible to add newlines _instead of_ via `<br>` tags.   Tables have limited newline support, hence usage of `<br>`.
* My understanding is folks generally prefer markdown _without_ HTML when possible.  In the highlighted paragraph, I don't _think_ the `<br>` tags are required.